src/core/lib/http/httpcli_security_connector.cc

```diff
@@ -192,10 +192,10 @@ class HttpRequestSSLCredentials : public grpc_channel_credentials {
   }
 
  private:
-  int cmp_impl(const grpc_channel_credentials* other) const override {
-    // TODO(yashykt): Check if we can do something better here
-    return QsortCompare(static_cast<const grpc_channel_credentials*>(this),
-                        other);
+  int cmp_impl(const grpc_channel_credentials* /* other */) const override {
+    // There's no differentiating factor between two HttpRequestSSLCredentials
```

Comment:
If it is the case, should we also create a singleton object for `HttpRequestSslCredentials`?

Reply:
Done